CHANGELOG.md

```diff
@@ -6,7 +6,117 @@ document.
 
 ## Unreleased / Beta / In Rust Nightly
 
-[1e5237f4...master](https://github.com/rust-lang/rust-clippy/compare/1e5237f4...master)
+[03a5b6b9...master](https://github.com/rust-lang/rust-clippy/compare/03a5b6b9...master)
+
+## Rust 1.88
+
+Current stable, released 2025-06-26
+
+[View all 137 merged pull requests](https://github.com/rust-lang/rust-clippy/pulls?q=merged%3A2025-05-01T16%3A52%3A57Z..2025-06-13T08%3A33%3A27Z+base%3Amaster)
```

Comment:
I'm surprised by the date range, I would have expected to find something close from:
- start: the time 1.87 was branched from master (March 28), adjusted to the time of the last sync before this (March 20), because everything before this went into 1.87
- end: the time 1.88 was branched from master (May 9), adjusted to the time of the last sync before this (May 1st)

Here, the date range is very different (May 1st to June 13 instead of March 28 to May 1st). It looks rather like the time span for Rust 1.89. Or am I wrong?

For example, it looks like the `swap_with_temporary` lint will be part of Rust 1.88 and is not listed below.

Reply:
Looks correct now -> resolving.